docs/modules/ROOT/pages/type-definitions/relationships.adoc

```diff
@@ -177,17 +177,56 @@ mutation CreateActor {
 }
 ----
 
-Note the selection of the `actorsConnection` field in order to query the `roles` relationship property.
+Note the selection of the `actedInMoviesConnection` field in order to query the `role` relationship property.
+
+It is also possible update nodes and to create duplicates of the same connection by specifying `createAsDuplicate: true` on connections that should not override existing ones.
+In this case, Tom Hanks also played Scrooge in the Polar Express:
+
+[source, graphql, indent=0]
+----
+mutation UpdateActor {
+    updatePeople(
+        where: { name: "Tom Hanks" }
+        connect: {
+            actedInMovies: [
+                {
+                    createAsDuplicate: true
```

Comment:
This doesn't match the `asDuplicate` implementation!

Reply:
Yeah this is because same as you I've not been sure on the arg name and have gone through a few different ones!

I've fixed this now but I'll leave this open for now in case you want to have more of a chat about the arg name

Reply:
Okaaaayyyyyy, so just taking a look at something which I think is quite similar, query direction.

In the `@relationship` directive, it is defined by `queryDirection` which accepts an enum of 4 possible values:

- `DEFAULT_DIRECTED` (default) 
- `DEFAULT_UNDIRECTED`
- `DIRECTED_ONLY`
- `UNDIRECTED_ONLY`

And then when querying, the option is called `directed`, which is a boolean. This argument is not present if `DIRECTED_ONLY` or `UNDIRECTED_ONLY` is specified in the directive.

I think the above is quite interesting, because it could take us back to an enum (but with 4 options) in the directive, and then a boolean argument which may or may not be present in `connect` operations based on the relationship directive.

So what if we had, in the `@relationship` directive, `allowDuplicates`, which accepts an enum of 4 values:

- `DEFAULT_NO` (current library default) - boolean option is present and false by default
- `DEFAULT_YES` - boolean option is present and true by default
- `ALWAYS` - boolean option is not present and duplicates are always made
- `NEVER` - boolean option is not present and duplicates are never made (technically current behaviour)

And then perhaps we just have `duplicate` as the boolean argument (as in the verb, not the adjective)? Let's discuss tomorrow, but absolutely we need to get this merged!